src/PKSim.CLI/PKSim.CLI.csproj

```diff
@@ -17,6 +17,7 @@
     <OutputPath>bin\$(Configuration)</OutputPath>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
     <NoWarn>1591, 3246</NoWarn>
+    <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
```

Comment:
That's required to run the CLI in net8. This is a vulnerability we have in OSPSuite, the use of the BinaryFormatter is discouraged.